src/usr/bin/google_set_multiqueue

```diff
@@ -76,10 +76,14 @@ function set_irq_range() {
     ((bind_cores_index++))
 
     # this is GVE's TX irq. See gve_tx_idx_to_ntfy().
-    echo "$core" > /proc/irq/"$tx_irq"/smp_affinity_list >&2
+    echo "$core" > "/proc/irq/${tx_irq}/smp_affinity_list"
 
     # this is GVE's RX irq. See gve_rx_idx_to_ntfy().
-    echo "$core" > /proc/irq/"$rx_irq"/smp_affinity_list >&2
+    echo "$core" > "/proc/irq/${rx_irq}/smp_affinity_list"
+
+    echo "tx_irq: ${tx_irq}, assigned irq core: $(cat "/proc/irq/${tx_irq}/smp_affinity_list")" >&2
```

Comment:
can you please move this message printout closer to where the command happen? tx right after tx and rx right after rx?